README.md

```diff
@@ -2,23 +2,26 @@
 
 [![Slack Status](http://slack.openpolicyagent.org/badge.svg)](https://slack.openpolicyagent.org) [![Build Status](https://github.com/open-policy-agent/opa/workflows/Post%20Merge/badge.svg?branch=main)](https://github.com/open-policy-agent/opa/actions) [![Go Report Card](https://goreportcard.com/badge/open-policy-agent/opa)](https://goreportcard.com/report/open-policy-agent/opa) [![CII Best Practices](https://bestpractices.coreinfrastructure.org/projects/1768/badge)](https://bestpractices.coreinfrastructure.org/projects/1768) [![Netlify Status](https://api.netlify.com/api/v1/badges/4a0a092a-8741-4826-a28f-826d4a576cab/deploy-status)](https://app.netlify.com/sites/openpolicyagent/deploys)
 
-The Open Policy Agent (OPA) is an open source, general-purpose policy engine that enables unified, context-aware policy enforcement across the entire stack.
+The Open Policy Agent (OPA) project is an open source, general-purpose policy engine that enables unified, context-aware policy enforcement across the entire stack.
```

Comment:
This is very much nitpick, but it reads a little weird to me that the "project" is a policy engine.. the project to me is all things OPA, like the policy engine, the community, etc. So I would keep this as it is. Maybe drop the leading "The" though, since no one ever says "The OPA" about OPA, or not as far as I ever heard :P